### PR TITLE
Fix publish container permissions issue/release 1.13.2

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-1415](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1415)

## Changes
* Fix broken publish script
* Update CI to allow SKIP_AUDIT flag to be set

### CODE REVIEW

File: bamboo/deploy-to-npm.sh [hunk 1]
```diff
@@ -7,7 +7,8 @@ export VERSION=$(jq --raw-output .version lerna.json)
 export NPM_TAG=$(node ./bamboo/npm-tag.js);
 
 echo "Publishing packages to NPM with version=${VERSION} and tag=${NPM_TAG}"
-lerna publish \
+export npm_config_unsafe_perm=true
```
Comment (laurenfrederick) [inline bamboo/deploy-to-npm.sh]: Comment this please.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [v.13.1] - 2019-07-22
+## [v1.13.2] - 2019-07-25
+
+- Re-release of v1.13.1 to fix broken npm packages.
+
+## [v1.13.1] - 2019-07-22
```
Comment (laurenfrederick) [inline CHANGELOG.md]: Please put a note in to not use 1.13.1, but to upgrade to 1.13.2

Comment (Jkovarik) [reply CHANGELOG.md]: e197f0842

File: bamboo/deploy-to-npm.sh (hunk 1 above)
Comment (Jkovarik) [reply bamboo/deploy-to-npm.sh]: e197f0842

**Pull request merged**